source/scripts/app.js

```diff
@@ -439,6 +413,58 @@ function populateRecipePage(recipeObj, fromSpoonacular) {
   }
 }
 
+/**
+ * Populates new recipes in the home page by retrieving new recipes from
+ * Spoonacular
+ * @function populateHomePage
+ */
+async function populateHomePage() {
+  "use strict";
+  let shadow = document.querySelector("home-page").shadowRoot;
+  const explore = shadow.getElementById("explore");
+
+  for (let i = 0; i < HOME_PAGE_NUM_RESULTS; ++i) {
+    const recipeCard = document.createElement("recipe-card");
+    explore.append(recipeCard);
+  }
+
+  let recipes = await spoonacular.getRandomRecipes(HOME_PAGE_NUM_RESULTS);
+  let recipeCards = explore.children;
+
+  for (let i = 0; i < recipeCards.length; ++i) {
+    let shadow = recipeCards[i].shadowRoot;
+    shadow.getElementById("recipe-id").textContent = recipes[i].id;
+    shadow.getElementById("recipe-card-title").textContent = recipes[i].title;
+    shadow.getElementById("recipe-card-image").src = recipes[i].image;
+  }
+
+  /**
+   * After clicking on "learn more" user is redirected to recipe form with all relevant info
+   * and an option to add to the cookbook (but NOT edit)
+   */
+
+  const dummy = {
+    title: "title",
+    author: "author",
+    cuisines: ["cuisine-0", "cuisine-1"],
+    readyInMinutes: 10,
+    image: "/source/images/pasta.jpg",
+    description: "description",
+    ingredients: ["ingredient-1", "ingredient-2"],
+    instructions: ["instruction-1", "instruction-2"],
+  };
+
+  //TODO change dummy to recipes[i] once needed
+  for (let i = 0; i < recipeCards.length; i++) {
+    let button = recipeCards[i].shadowRoot.getElementById("recipe-info-button");
+
+    button.addEventListener("click", populateRecipePage(dummy, true));
```

Comment:
```suggestion

```

Reply:
Does combining the two event listeners work?

Reply:
I kept getting a jshint error when I combined both, so I just separated them into two. The functionality is still the same